### DISCUSSION TITLE
Reinforcement learning algorithms are incorporated into the wind farm model

### DISCUSSION THREAD

**Discussion by hao-user** [NatLabRockies/floris#1012] [Q&A]
I learned how to apply reinforcement learning to wind farm models.Can someone share some simple journeys for me to learn from?
Or give me some advice on how to study。
thanks very much

**Comment by paulf81** [comment]
I found for example this literature review on the subject: https://www.sciencedirect.com/science/article/pii/S2095034923000466

**Reply by hao-user** [reply]
Thank you for your reply, please do you have any simple program examples for me to learn, thank you very much

**Reply by paulf81** [reply]
no sorry, the included examples in the [documentation](https://nrel.github.io/floris/)include demonstrations of some optimizations, but not reinforcement learning
